### PR TITLE
LPS-149330 | LPS-149339

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issues: 
https://issues.liferay.com/browse/LPS-149330
https://issues.liferay.com/browse/LPS-149339

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Review by rodrigopaulino — COMMENTED** [review]
Considerations were made directly to @carolmariaabb.

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by rodrigopaulino — APPROVED** [review]
Great job! Thanks a lot @carolmariaabb 
👏🏻

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx [hunk 1]
```diff
@@ -87,17 +87,14 @@ export default function ObjectFieldFormBase({
...
 							value: 'jpeg, jpg, pdf, png',
 						},
 						{
 							name: 'fileSource',
-							required: true,
 							value: 'userComputer',
 						},
 						{
 							name: 'maximumFileSize',
-							required: true,
```
Comment (carolmariaabb) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx:100]: @miltonmc

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/index.d.ts [hunk 2]
```diff
@@ -37,7 +37,6 @@ interface ObjectField {
 
 interface ObjectFieldSetting {
 	name: ObjectFieldSettingName;
-	required: boolean;
```
Comment (carolmariaabb) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/index.d.ts:40]: @miltonmc

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
Forwarded https://github.com/liferay-forms/liferay-portal/pull/1539

**Pull request closed without merging**